### PR TITLE
Remove Reanimated 1 Transitions API usages

### PR DESCRIPTION
**Pull request by jkadamczyk**
Fixes RNBW-3703
Fixes RNBW-3702
Fixes RNBW-3700
Fixes RNBW-3698
Fixes RNBW-3697
Fixes RNBW-3696
Fixes RNBW-3695
Fixes RNBW-3692

## What changed (plus any additional context for devs)
* Removed unused components
* Replaced some of Reanimated 1 Transitions API usages with Reanimated 2 Layout Animations
* Removed some of Reanimated 1 Transitions API usages as they weren't working and were generally unused, not neccesary

## PoW (screenshots / screen recordings)
Clear Input Decorator:
iOS: https://streamable.com/cg1f8y
Android: https://streamable.com/nxliin

Send Empty state:
iOS: https://streamable.com/6cub76
Android: https://streamable.com/pdicda

Wallet Switch Modal:
iOS: https://streamable.com/o61xwf
Android: https://streamable.com/rxpbuo

Add Cash Status:
iOS: https://streamable.com/d9r6ea
Android: Different provider web UI

## Dev checklist for QA: what to test
Check following components transitions:
* The ones shown in recordings above
* Currency selection in swap modal, there was defined transition for hiding the skeleton state, but it wasn't working, now there's no transition
* Check AddCashStatus component transition in, can be checked by mocking isPaymentComplete in code and switching state with setTimeout
* KeyboardFixedOpenLayout – removed the layout change transition as it wasn't working in any of the cases on iOS or Android – can be seen in exchange modal check if it looks good
* WalletList – present in Wallet change sheet, the animation was broken, now it should crossfade from skeleton to wallet list almost immediately when the sheet is opened

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [x] Added e2e tests? if not please specify why – aniamtions can't be tested reliably
- [x] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Comment by skylarbarrera** [comment]
adding `needs QA` since this is pretty user facing

**Review by brunobar79 — APPROVED** [review]
I don't know shit about reanimated so I'm approving to prevent blocking

File: src/components/change-wallet/WalletList.js [hunk 1]
```diff
@@ -237,9 +225,16 @@ export default function WalletList({
               />
             </WalletListFooter>
           )}
-        </Fragment>
+        </WalletsContainer>
       ) : (
-        <EmptyWalletList />
+        <Animated.View
+          entering={FadeIn.easing(Easing.out(Easing.ease)).duration(0.001)}
```
Comment (terrysahaidak) [inline src/components/change-wallet/WalletList.js:231]: Do we really need this?

File: src/components/send/SendEmptyState.js [hunk 2]
```diff
@@ -56,7 +38,15 @@ const SendEmptyState = () => {
   );
 
   if (android) {
-    return <View style={{ alignItems: 'center', flex: 1 }}>{icon}</View>;
+    return (
+      <Animated.View
+        entering={keyframe.duration(duration)}
+        exiting={FadeOut.duration(duration).easing(Easing.in(Easing.ease))}
+        style={{ alignItems: 'center', flex: 1 }}
```
Comment (terrysahaidak) [inline src/components/send/SendEmptyState.js:45]: Can you move it to styles?

**Review by terrysahaidak — APPROVED** [review]
I would want @christianbaroni to look at this since we had pretty much all of the transitions not working before these changes.

**Review by estebanmino — APPROVED** [review]

**Review by ibrahimtaveras00 — APPROVED** [review]

**Comment by jkadamczyk** [comment]
Removing the label since it was looked at by christian and addressed by me

**Pull request merged**